apps/api/src/app/workflows-v1-deprecated/dto/change-workflow-status-request.dto.ts

```diff
@@ -1,6 +1,9 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsBoolean, IsDefined } from 'class-validator';
 
+/**
+ * @deprecated use dto's in /workflows directory
+ */
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** nice comment 🙏‏

Reply:
yei!